druid/src/env.rs

```diff
@@ -76,6 +78,18 @@ pub enum Value {
     String(String),
 }
 
+/// Either a concrete `T` or a [`Key<T>`] that can be resolved in the [`Env`].
+///
+/// This is a way to allow widgets to interchangeably use either a specific
+/// value or a value from the environment for some purpose.
+///
+/// [`Key<T>`]: struct.Key.html
+/// [`Env<T>`]: struct.Env.html
```

Comment:
oops my mistake, this is wrong, should just be `Env`.

Reply:
not sure what you're referring to